components/spicedb/schema/schema.yaml

```diff
@@ -17,7 +17,7 @@ schema: |-
 
       permission make_admin = installation->admin + organization->installation_admin
 
-      // administrate is for changes such as blocking or verifiying, i.e. things that only admins can do on user
+      // administrate is for changes such as blocking or verifying, i.e. things that only admins can do on user
```

Comment:
Let's not change this if not needed, I'm 95% sure it will trigger a spicedb rollover which I'd like to avoid, because it has the tendency to take long time to cycle and not behave well in the meantim.

Reply:
Oh right, good point! Will revert to not cause unnecessary trouble

Reply:
Addressed in [`bda0663` (#20565)](https://github.com/gitpod-io/gitpod/pull/20565/commits/bda06630679c0ec97edae87e969cf5d9ad453337)